pkg/koordlet/qosmanager/plugins/memoryevict/memory_evict.go

```diff
@@ -164,24 +166,28 @@ func (m *memoryEvictor) killAndEvictBEPods(node *corev1.Node, podMetrics map[str
 	bePodInfos := m.getSortedBEPodInfos(podMetrics)
 	message := fmt.Sprintf("killAndEvictBEPods for node, need to release memory: %v", memoryNeedRelease)
 	memoryReleased := int64(0)
-
-	var killedPods []*corev1.Pod
+	hasKillPods := false
 	for _, bePod := range bePodInfos {
 		if memoryReleased >= memoryNeedRelease {
 			break
 		}
 
+		ok := m.evictor.EvictPodIfNotEvicted(bePod.pod, node, resourceexecutor.EvictPodByNodeMemoryUsage, message)
+		if m.onlyEvictByAPI && !ok {
```

Comment:
please add some error logs if the eviction fails, so does the cpu evict

Reply:
EvictPodIfNotEvicted->evictPod already has a log.
Do you need it here?